app/assets/src/components/visualizations/table/BaseTable.jsx

```diff
@@ -14,11 +13,21 @@ import BasicPopup from "~/components/BasicPopup";
 import Checkbox from "~ui/controls/Checkbox";
 import MultipleDropdown from "~ui/controls/dropdowns/MultipleDropdown";
 import PlusIcon from "~ui/icons/PlusIcon";
+import SortIcon from "~ui/icons/SortIcon";
 import { humanize } from "~/helpers/strings";
 import { logAnalyticsEvent } from "~/api/analytics";
 
 import cs from "./base_table.scss";
 
+const addDefaultsToColumns = (columns, defaultColumnWidth) => {
```

Comment:
Why define this outside? 
YOu can just keep the setDefaults function but change its signature.